test/integration/machine-learning.spec.ts

```diff
@@ -101,6 +105,58 @@ describe('admin.machineLearning', () => {
         });
     });
 
+    it('creates a new Model with valid AutoML TFLite ModelFormat', function () {
+      const modelOptions: admin.machineLearning.ModelOptions = {
+        displayName: 'node-integ-test-create-automl',
+        tags: ['tagAutoml'],
+        tfliteModel: { automlModel: 'this will be replaced below' }
+      };
+      this.timeout(60000); // Allow up to 60 seconds for this test.
+      return getAutoMLModelReference()
+        .then((automlRef: string) => {
+          if (!automlRef) {
+            this.skip();
+            return;
+          }
+          modelOptions.tfliteModel!.automlModel = automlRef;
+          return admin.machineLearning().createModel(modelOptions)
+            .then((model) => {
+              return model.waitForUnlocked(55000)
+                .then(() => {
+                  scheduleForDelete(model);
+                  verifyModel(model, modelOptions);
+                });
+            });
+        });
+    });
+
+    // Waiting for BE fix.
```

Comment:
Let's implement this test case the way it should be, and just skip the whole thing for now by putting `xit.` on it.

Reply:
It is implemented how it should be. I skipped it with it.skip is xit better?

Reply:
Then I don't think the description of the test is correct. I don't see any code that handles a rejection in the body of the test.